proj/src/reducers/projects/projectsReducer.js

```diff
@@ -7,11 +7,7 @@ const initialState = {
     displayEditModal: false,
     displayMode: 'list',
     currProject: null,
-    projects: {
-        1: {id: 1, name: 'Project 1'},
-        2: {id: 2, name: 'Project 2'},
-        3: {id: 3, name: 'Project 3'},
-    }
+    projects: {}
```

Comment:
this means you still need to access 'state.projects.projects' in connected components. you can just remove the inner 'projects' property altogether. you will also have to fix the appropriate places in the reducer to set directly in the state and not in state.projects